content/introduction/what-is-libp2p.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "What is libp2p?"
+menu:
+    introduction:
+      weight: 1
+---
+
+Good question! The one-liner pitch is that libp2p is a modular system of *protocols*, *specifications* and *libraries* that enable the development of peer-to-peer network applications.
+
+<!--more-->
+
+## Peer-to-peer basics
+
+There's a lot to unpack in that one-liner! Let's start with the last bit, "peer-to-peer network applications." You may be here because you're knee-deep in development of a peer-to-peer system and are looking for help.  Likewise, you may be here because you're just exploring the world of peer-to-peer networking for the first time.  Either way, we ought to spend a minute defining our terms upfront, so we can have some shared vocabulary to build on.
+
+
+
+A [peer-to-peer network][definition_p2p] is one in which the participants (referred to as [peers][definition_peer] or nodes) communicate with one another directly, on more or less "equal footing". This does not necessarily mean that all peers are identical; some may have different roles in the overall network.  However, one of the defining characteristics of a peer-to-peer network is that they do not require a priviliged set of "servers" which behave completely differently from their "clients", as is the case in the the predominant [client / server model][definition_client_server].
+
+
+
+Because the definition of peer-to-peer networking is quite broad, many different kinds of systems have been built that all fall under the umbrella of "peer-to-peer".  The most culturally prominent examples are likely the file sharing networks like bittorrent, and, more recently, the proliferation of blockchain networks that communicate in a peer-to-peer fashion.
+
+## What problems can libp2p solve?
```

Comment:
I suggest we add a note here similar to the ["IPFS and Filecoin" FAQ](https://filecoin.io/faqs/#what-is-the-connection-between-ipfs-and-filecoin) that describes how libp2p and IPFS work together and solve component parts of the problem of decentralizing the web - designed to work really well together. =]